packages/react-charting/src/components/HorizontalBarChartWithAxis/__snapshots__/HorizontalBarChartWithAxisRTL.test.tsx.snap

```diff
@@ -0,0 +1,1001 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`HorizontalBarChartWithAxis - mouse events Should render callout correctly on mouseover 1`] = `
```

Comment:
is the content for the 2 snapshots same?

Reply:
nope, snap for test and RTL test are different